samples/04_gis_analysts_data_scientists/creating_raster_information_product_using_raster_analytics.ipynb

```diff
@@ -20,15 +20,15 @@
   },
```

Comment:
<p>Can we update this comment to something like - 'Here we're getting the multispectral landsat imagery item', as we are changing search method to get</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>